src/compat/posix/fs/stat.c

```diff
@@ -6,7 +6,6 @@
  */
 
 #include <sys/stat.h>
-
```

Comment:
Please, avoid such changes (i.e. you don't actually change anything but just fix formatting), it's not related to your branch and makes it harder to figure out which files were actually changed.

Reply:
undo all useless commits